convert/convert.go

```diff
@@ -194,6 +196,78 @@ func bedMidpoint(b bed.Bed) int {
 	return (b.ChromEnd + b.ChromStart) / 2
 }
 
+func ThreeWayFaToVcf(f []fasta.Fasta, chr string, out *fileio.EasyWriter, substitutionOnly bool, retainN bool) {
+	var currRefPos, currAlnPos int = 0, 0
+	// var insertion1, insertion2, deletion1, deletion2 bool = false, false, false, false
```

Comment:
If we're committing to only handling substitutions, you can delete the commented out lines related to INDEL logic